app/controllers/bids_controller.rb

```diff
@@ -31,13 +30,14 @@ def new
   end
 
   def confirm
```

Comment:
There should really only be rest methods in a controller. Usually when you need non-rest methods, there is a missing concept that has its own subset of rest needs.